.isort.cfg

```diff
@@ -2,6 +2,6 @@
 force_single_line=true
 force_sort_within_sections=true
 ignore_whitespace=true
-known_third_party=browsermobproxy,cloudstorage,google.appengine,jinja2,mapreduce,mutagen,pipeline,pylatexenc,pylint,requests,selenium,skulpt,webapp2,webapp2_extras,webtest,yaml
+known_third_party=browsermobproxy,cloudstorage,google.appengine,google.oauth2,jinja2,mapreduce,mutagen,pipeline,pylatexenc,pylint,requests,selenium,skulpt,webapp2,webapp2_extras,webtest,yaml
```

Comment:
Where do we use `google.oauth2` on develop btw? [Alsking this as #10654 is reading this lib in requirements.in]

Also, considering the changes in other file in this PR, are we expecting this to be `google.protobuf`?

Reply:
I just added `google.protobuf`. `google.oauth2` is about to be added by #10654, but I'll remove it from this PR since it's not in develop yet